tests/system/action/organization/test_initial_import.py

```diff
@@ -1,17 +1,33 @@
+from typing import Any, Dict, List
+
+from datastore.migrations import setup
+
+from migrations import assert_migration_index
+from migrations.migrate import load_migrations
+from openslides_backend.shared.util import INITIAL_DATA_FILE, get_initial_data_file
 from tests.system.action.base import BaseActionTestCase
 
 
+def get_backend_migration_index(migrations: List) -> int:
+    backend_migration_index = 1
+    for migration_class in migrations:
+        backend_migration_index = max(
+            backend_migration_index, migration_class().target_migration_index
+        )
+    return backend_migration_index
```

Comment:
remove and use the method in the migration module once https://github.com/OpenSlides/openslides-backend/pull/677 is merged (which should be any minute...)